sql/src/main/java/org/apache/druid/sql/http/SupportedEnginesResponse.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.http;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Set;
+
+public class SupportedEnginesResponse
+{
+  private final Set<String> supportedEngines;
+
+  @JsonCreator
+  public SupportedEnginesResponse(@JsonProperty("supportedEngines")Set<String> supportedEngines)
```

Comment:
Should just be `engines` IMO, keep it simple. Also, please make each engine an object rather than a string. Someday we may want to add additional per-engine properties, so an object makes this possible.